remixer_example/src/main/java/com/google/android/apps/remixer/RemixerApplication.java

```diff
@@ -35,5 +36,7 @@
   public void onCreate() {
     super.onCreate();
     RemixerInitialization.initRemixer(this);
+    Remixer.getInstance().setSynchronizationMechanism(
+        new LocalStorage(getApplicationContext(), "Remixer"));
```

Comment:
what happens if the client does not provide this line?

I kind of expect that the default sync mech would include this. I can also see why that's not wanted. What do you think?